config/configgrpc/configgrpc.go

```diff
@@ -90,12 +95,27 @@ type ClientConfig struct {
 	Auth *configauth.Authentication `mapstructure:"auth"`
 }
 
+// NewDefaultClientConfig() creates a new ClientConfig with any default values set
+func NewDefaultClientConfig() *ClientConfig {
+	return &ClientConfig{
```

Comment:
This function will need to call configauth.NewDefaultAuthentication once it exists.